src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/BasicExpressionCompilerTest.vbproj

```diff
@@ -99,5 +99,8 @@
   <ItemGroup>
     <Folder Include="My Project\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
```

Comment:
@jaredpar I think you're missing a word, but I believe the behavior is by design. Test projects are supposed to have this to my understanding, but I'm not an expert in this area.